Vizzuality/half-earth-v3
# Feature/translate precalculated strings

## Use transifex to translate precalculated string
### Description
Country names and strings on Aoi and protected areas tables are now stored on a constants file and uploaded to transifex from them. 

Important. This solution has to be maintained if these fields change!

### Testing instructions
Go to Spain AOI and check if the country name the first fields on the sidebar and the data on the protected areas table are translated when we change to spanish

### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-490?atlOrigin=eyJpIjoiMzM5MGI2YzBhOWU3NGNkNGIwMDI0ZTJhZjQ2ZjE4OGMiLCJwIjoiaiJ9
